[BUG] Document the feature that allows users to add alerts to cases (#758)
elastic/security-docs: docs/detections/alerts-ui-manage.asciidoc

* In the Alerts table, select all the alerts you want to change, and then select
*Take action* -> *Close selected*, *Open selected*, or *Mark in progress*.

[float]
[[signals-to-cases]]
=== Add alerts to cases

> From the Alerts table, you can attach one or more alerts to a case by using the *Add to case* button. From here, you can choose to <<signals-to-new-cases, add the alert to a new case>> or <<signals-to-existing-cases, attach it to an existing one>>. There is no limit to the number of alerts that you can add and you can choose alerts from any rule type - custom or pre-built. If the case has been configured to sync its status with associated alerts, the alert's status will update after it's been added. If this setting isn't enabled, the alert's status will not change after you've added it.

Reviewer: > If the case has been configured to sync its status with associated alerts, the alert's status will update after it's been added. If this setting isn't enabled, the alert's status will not change after you've added it.

Not sure if it's necessary but should we add a sentence about when the `status` would be updated? Like:
`If the case has been configured to sync it's status with associated alerts, the alert's status will update whenever the case's status is changed`

That might be redundant though.

Reviewer 2: Yes, I think it should be clear that the alert's status will change when you change the case's status and sync is on.

> NOTE: After you have added an alert to a case, you can only remove it through the the Elastic Security Cases API.

Reviewer 3: ```suggestion
NOTE: Once you've added an alert to a case, you can only remove it through the the <<cases-api-overview, Elastic Security Cases API>>.
```

[role="screenshot"]
image::images/add-alert-to-case.gif[width=80%][height=80%][Shows how to add an alert to a case]

[float]
[[signals-to-new-cases]]
==== Add an alert to a new case

> Follow these steps to add an alert to a new case:

Reviewer 3: ```suggestion
To add an alert to a new case:
```

. Select **Add to case -> Add to a new case**.

> . In the flyout, give your case a name, add relevant tags, and include a case description.

Reviewer 3: ```suggestion
. In the **Create a new case** pane, give your case a name, add relevant tags, and include a case description.
```

> . Specify whether you want to sync the statuses of associated alerts. This setting is enabled by default to enforce asynchronous communication between alerts and cases and can be toggled on or off at any time.

Reviewer: > This setting is enabled by default to enforce asynchronous communication between alerts and cases and can be toggled on or off at any time.

nit: _asynchronous communication_ makes me think of bidirectional communication between cases and alerts (maybe that's just an assumption though). When the setting is enabled Cases will control the status of the alerts. Alerts don't really communicate back to Cases.

Maybe something like:

`This setting is enabled by default to allow Cases to control the state of attached alerts' status. Whenever the Case status changes the status of the attached alerts will be updated to match it. It can be toggled on or off at any time.`

Reviewer 2: I agree with @jonathan-buttner. It also makes me think the same.

> . Specify whether you want to sync the statuses of associated alerts. This setting can be toggled on or off at any time, but is enabled by default to allow Cases to control the statuses of attached alerts. If enabled, the alert's status updates any time the case's status is modified.

Reviewer 3: ```suggestion
. Specify whether you want to sync the status of associated alerts. It is enabled by default; however, you can toggle this setting on or off at any time. If it remains enabled, the alert's status updates whenever the case's status is modified.
```

. Select a connector. If you've previously added one, that connector displays as the default selection. Otherwise, the default setting is `No connector selected`.
. Click *Create case* after you've completed all of the required fields. A notification message confirming the case was successfully created displays. Click the link inside the notification or go to the Cases page to view your case.

[role="screenshot"]
image::images/add-alert-to-new-case.png[Shows how to add an alert to an existing case]

[float]
[[signals-to-existing-cases]]

> ==== Add an alert to an existing case

Reviewer 3: Let's make the steps a numbered procedure - (even if it's two steps) for consistency.

> To attach an alert to an existing case, select **Add to case -> Add to existing case**. From the Select case pane, click on a case to choose it. Following your selection, a confirmation message displays with an option to view the updated case. Click on the link in the notification or go to the Cases page to view the case's details.

Reviewer 3: ```suggestion
To attach an alert to an existing case, select **Add to case -> Add to existing case**.  In the **Select case** pane, select the appropriate case for which to attach an alert. A confirmation message displays with an option to view the updated case. Click on the link in the notification or go to the Cases page to view the case's details.
```
